src/fmgc/src/components/ReadySignal.ts

```diff
@@ -0,0 +1,22 @@
+import { FlightPlanManager } from '@fmgc/wtsdk';
+import { FmgcComponent } from './FmgcComponent';
+
+export class ReadySignal implements FmgcComponent {
+    private baseInstrument: BaseInstrument = null;
+
+    private updateThrottler = new A32NX_Util.UpdateThrottler(1000);
+
+    init(baseInstrument: BaseInstrument, _flightPlanManager: FlightPlanManager): void {
+        this.baseInstrument = baseInstrument;
+    }
+
+    update(deltaTime: number): void {
+        if (this.updateThrottler.canUpdate(deltaTime) !== -1
+            && this.baseInstrument.getGameState() === GameState.ingame
+            && SimVar.GetSimVarValue('L:A32NX_IS_READY', 'number') !== 1) {
```

Comment:
We could probably cache the set state in a member variable to save reading the A32NX_IS_READY simvar every frame?

Reply:
Don’t forget there is an update throttler above…

Reply:
My bad, I did miss that! This is fine then. 👍